modules/openapi/src/alloy/openapi/package.scala

```diff
@@ -32,24 +31,10 @@ package object openapi {
 
   def convertWithConfig(
       model: Model,
-      allowedNS: Option[Set[String]],
+      service: Shape,
```

Comment:
Won't this be binary breaking? Perhaps we can add this in addition to the previous one? I know there are other overloads, but I just don't know if it is worth breaking bin-compat when it should be fairly easy to avoid

Reply:
Ahhh sorry never mind, I see below that the diff is just weird and you did keep the prior variant